lib/devise/multi_email/models/confirmable.rb

```diff
@@ -45,10 +45,16 @@ module ConfirmableExtensions
                  :confirmation_token, :confirmed_at, :confirmed_at=, :confirmation_sent_at, :confirm, :confirmed?, :unconfirmed_email,
                  :reconfirmation_required?, :pending_reconfirmation?, to: Devise::MultiEmail.primary_email_method_name, allow_nil: true
 
+        %i{skip_confirmation! skip_confirmation_notification! skip_reconfirmation!}.each do |key|
```

Comment:
These methods are intentionally delegated to primary email only, don't think we should do it on all emails. That breaks the current contract.

Reply:
But non-primary emails also need confirmation or skip_confirmation, don't they?

Reply:
looking at the methods above (`confirmed_at`, `confirmed`,  etc), they are all delegated to primary email. If we do only do `skip` for all emails, that certainly leads to lots of confusions to users. So I'd let users handle them manually.

May I know the reason behind this change and possibly your use case?